asreview/logging.py

```diff
@@ -1,230 +1,35 @@
-import copy
-import json
-from collections import OrderedDict
-from datetime import datetime
-from pathlib import Path
+import os
 
-import asreview
-import numpy as np
-from asreview.settings import ASReviewSettings
 
+def get_logger_class(fp):
```

Comment:
`fp=None`

Reply:
Are you sure about this? To me it doesn't make much sense for the DictLogger to be a default choice, or at least I feel like that is not really appropriate in this case.